product_docs/docs/pgd/5/durability/administering.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Administering
+navTitle: Administering
+---
+
+When running a PGD cluster with Group Commit, there are some things you need to
+be aware of when administering the system, such as how to safely shut down and 
+restart nodes.
+
+## Planned shutdown and restarts
+
+When using Group Commit with receive confirmations, take care
+with planned shutdown or restart. By default, the apply queue is processed
+prior to shutting down. However, in the `immediate` shutdown mode, the queue
+is discarded at shutdown, leading to the stopped node "forgetting"
+transactions in the queue. A concurrent failure of the origin node can
+lead to loss of data, as if both nodes failed.
+
+To ensure the apply queue gets flushed to disk, use either
+`smart` or `fast` shutdown for maintenance tasks. This approach maintains the
```

Comment:
What's the difference between these two shutdown modes?

Reply:
:shrug: No idea.

Reply:
This refers to the various [ways a PostgreSQL server can be shut down](https://www.postgresql.org/docs/current/server-shutdown.html). So it may make sense to either link to that document or give a hint that we're talking about PostgreSQL's different shutdown methods here (and not some PGD-specific way).